ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/http-client/common.main/module.xml

```diff
@@ -19,24 +19,24 @@
   ~ You should have received a copy of the GNU Lesser General Public
   ~ License along with this software; if not, write to the Free
   ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
-  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  ~ 2110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
 
-<module xmlns="urn:jboss:module:1.9" name="org.wildfly.naming-client">
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.http-client.common">
 
     <resources>
-        <artifact name="${org.wildfly:wildfly-naming-client}"/>
+        <artifact name="\${org.wildfly.wildfly-http-client:wildfly-http-client-common@module.jakarta.suffix@}"/>
```

Comment:
You can just make this:

${org.wildfly:wildfly-naming-client-jakarta}

The complex string replacement syntax let us produce both an EE 8 and an EE 9+ server with the same module.xml file, but we don't need to do that any more.